### DISCUSSION TITLE
PIV authentication and sudo

### DISCUSSION THREAD

**Discussion by andreas65m** [NoMoreFood/putty-cac#146] [General]
Hello,

I followed these instructions https://www.idmanagement.gov/implement/scl-ssh/ 
and enabled the login via smartcard 
I have to use sudo for certain tasks, but at this point I am asked for my password.
if I have to enter my password in this case, there is not much point in logging in via PIV
Is there a way to use my smartcard even with sudo ( I can't immagine how ) or a way to prevent the server to ask my password?

Happy 2025
Andreas

**Comment by NoMoreFood** [comment]
You can certainly change sudo to not prompt you for a password.  Just google "visudo password".  The most common thing I've seen is simply using a different account entirely for administrative tasks and then using SSH forwarding to authenticate to that account instead.
